### ISSUE TITLE
-Wsometimes-uninitialized in drivers/gpu/drm/amd/powerplay/vega20_ppt.c

### ISSUE THREAD

**Issue by nathanchance** [ClangBuiltLinux/linux#425]
```
drivers/gpu/drm/amd/amdgpu/../powerplay/vega20_ppt.c:456:2: warning: variable 'num_of_levels' is used uninitialized whenever '?:' condition is false [-Wsometimes-uninitialized] 
```

**Comment by nathanchance** [comment]
Patch sent: https://lore.kernel.org/lkml/20190320005842.25383-1-natechancellor@gmail.com/

**Comment by nathanchance** [comment]
Patch accepted: https://cgit.freedesktop.org/~agd5f/linux/commit/?id=5ea8b4725f42c22c37b370eebb5d33b4236f5301

**Comment by nathanchance** [comment]
Fixed in `next-20190322`: https://git.kernel.org/next/linux-next/c/5ea8b4725f42c22c37b370eebb5d33b4236f5301
